### PR TITLE
Show previous appeals

### LINKED ISSUE #37: Show previous appeals in a table
Previous appeals by user and by ip should be listed on the appeal directly, not including itself.

Searching for account appeals only has to worry about appealfor in appeals table.
Searching for other IP appeals has to worry about both appealfor and hiddenip in the appeals table. It would also be prudent to search in the logs table under:
`{ user = 0, objecttype = appeal, action = create, ip = {ip} }`

Can I have you do this @supertassu ? I'm also going to add you to the project so we can add you to assign.

### PR DESCRIPTION
**Pull request by dqwiki**
Closes #37.
Stolen from #45 to enable proper code review. Only merge when cleanup-appeal-code is approved.

### CODE REVIEW

File: resources/views/appeals/appeal.blade.php
Comment (dqwiki) [inline resources/views/appeals/appeal.blade.php]: @supertassu Is there any practical change here?

File: resources/views/appeals/appeal.blade.php
Comment (dqwiki) [inline resources/views/appeals/appeal.blade.php]: Again here, any practical change?

**Review by dqwiki — COMMENTED** [review]
It's going to be more practical admins to have previous appeals running near or at the top of the page. Admins don't want to spend the extra time to look at the bottom for it. It'll surely become a issue in the future. Therefore I'll leave it to you to find a proper spot.

**Comment by dqwiki** [comment]
I have rebased this on master based on the other changes I already approved. I would also like to request a screenshot of this change before approval.

File: resources/views/appeals/appeal.blade.php
Comment (supertassu) [reply resources/views/appeals/appeal.blade.php]: Only practical change here is to wrap the input in `.form-group` and make text area `.form-control`

File: resources/views/appeals/appeal.blade.php
Comment (supertassu) [reply resources/views/appeals/appeal.blade.php]: As above

**Comment by supertassu** [comment]
@dqwiki Here's what it looks on b708e6420e52a032a7786897c87cc9dce731d0fe

image

**Review by supertassu — APPROVED** [review]

**Pull request merged**
